### PR TITLE
added a component in UI-common for choosing a vehicle AC or Non Ac

### PR DESCRIPTION
**Pull request by saireddy2610**
## Type of Change

- [ ] Bugfix
- [x
Screenshot 2023-03-23 at 5 07 12 PM
] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

### Additional Changes

- [ ] This PR modifies the database schema (database migration added)
- [ ] This PR modifies dhall configs/environment variables
- [ ] This PR contains API breaking changes

## Checklist

- [ ] I formatted the code and addressed linter errors `./dev/format-all-files.sh`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable

### CODE REVIEW

File: Frontend/ui-common/src/Components/ChooseVehicle/View.purs [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+        , padding state.secondaryImage.padding
+        , margin state.secondaryImage.margin
+        ][  imageView
+          [ imageWithFallback state.secondaryImage.imageUrl
+          , height state.secondaryImage.height
+          , width state.secondaryImage.width
+          , clickable true
+          , visibility state.secondaryImage.visibility
+          ]  
+
```
Comment (Rohit4997) [inline Frontend/ui-common/src/Components/ChooseVehicle/View.purs:137]: Remove extra space

File: Frontend/ui-common/src/Components/ChooseVehicle/View.purs [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+priceDetails push state = 
+ linearLayout
+  [ height WRAP_CONTENT
+  , width WRAP_CONTENT
+  , orientation HORIZONTAL
+  ][
+    textView $
+        [ width WRAP_CONTENT
+        , height WRAP_CONTENT
+        , text $ ("₹" <> state.cornerText.text)
```
Comment (Rohit4997) [inline Frontend/ui-common/src/Components/ChooseVehicle/View.purs:116]: Use either $ or ()

File: Frontend/ui-common/src/Components/ChooseVehicle/View.purs [hunk 3]
```diff
@@ -0,0 +1,141 @@
+module Components.ChooseVehicle.View where
+
+import Components.ChooseVehicle.Controller (Action(..), Config)
+import Components.PrimaryButton as PrimaryButton
+import Data.Maybe (fromMaybe)
+import Data.String (Pattern(..), split, length)
```
Comment (Rohit4997) [inline Frontend/ui-common/src/Components/ChooseVehicle/View.purs:6]: Remove this Import statement

File: Frontend/ui-common/src/Components/ChooseVehicle/View.purs [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+import Prelude (Unit,($),(<>),const,pure,unit)
+import PrestoDOM (Gravity(..), Length(..), Margin(..), Orientation(..), Padding(..), PrestoDOM, Visibility(..),  background, color, cornerRadius,  fontStyle,  gravity, height, imageView, letterSpacing, lineHeight, linearLayout, margin, onClick, orientation, padding, stroke, text, textSize, textView, visibility, weight, width, clickable, imageWithFallback , imageUrl)
+import PrestoDOM.Properties (cornerRadii)
+import PrestoDOM.Types.DomAttributes (Corners(..))
+import Screens.Types (Stage(..))
+import Styles.Colors as Color
+import Font.Size as FontSize
+import Font.Style as FontStyle
+import Common.Types.App
+
```
Comment (Rohit4997) [inline Frontend/ui-common/src/Components/ChooseVehicle/View.purs:22]: @saireddy2610  Please revisit through the import statement, and remove unused

Comment (saireddy2610) [reply Frontend/ui-common/src/Components/ChooseVehicle/View.purs:22]: kk

File: Frontend/ui-common/src/Components/ChooseVehicle/View.purs [hunk 5]
```diff
@@ -0,0 +1,141 @@
+module Components.ChooseVehicle.View where
+
+import Components.ChooseVehicle.Controller (Action(..), Config)
+import Components.PrimaryButton as PrimaryButton
```
[the remark this answers is not in the document]
Comment (saireddy2610) [reply Frontend/ui-common/src/Components/ChooseVehicle/View.purs:4]: Done

File: Frontend/ui-common/src/Components/ChooseVehicle/View.purs (hunk 2 above)
Comment (saireddy2610) [reply Frontend/ui-common/src/Components/ChooseVehicle/View.purs:116]: done

File: Frontend/ui-common/src/Components/ChooseVehicle/View.purs (hunk 1 above)
Comment (saireddy2610) [reply Frontend/ui-common/src/Components/ChooseVehicle/View.purs:137]: done

File: Frontend/android-native/app/src/user/res/drawable/ny_ic_sedan_white.png
Comment (Rohit4997) [inline Frontend/android-native/app/src/user/res/drawable/ny_ic_sedan_white.png:1]: There assets we have already added with the names `ic_suv_ac` and `ic_vehicle_front` can you use that

**Pull request closed without merging**